io.github.nyre221.kiview.json

```diff
@@ -0,0 +1,44 @@
+{
+   "id": "io.github.nyre221.kiview",
+   "runtime": "org.kde.Platform",
+   "runtime-version": "6.6",
+   "sdk": "org.kde.Sdk",
+   "base": "io.qt.qtwebengine.BaseApp",
+   "base-version": "6.6",
+   "command": "kiview",
+   "build-options": {
+   "env": [
+      "QTWEBENGINE_DISABLE_SANDBOX=1"
+    ]
+  },
+
+   "finish-args": [
+      "--share=ipc",
+      "--device=dri",
+      "--socket=fallback-x11",
+      "--socket=wayland",
+      "--env=QML_IMPORT_PATH=/app/qml" ,
+      "--socket=pulseaudio",
+      "--socket=session-bus",
+      "--filesystem=host:ro",
+      "--filesystem=/tmp",
```

Comment:
Why is this necessary?

Reply:
--socket=pulseaudio: to play audio/video files
--socket=session-bus: 

> It is used to obtain the list of all Dolphin windows and the name of their interface.
> Example in bash:
> $dbus-send --session --print-reply --type=method_call --dest=org.freedesktop.DBus /org/freedesktop/DBus org.freedesktop.DBus.ListNames | grep -i dolphin 
> $  string "org.kde.dolphin-13165"

Kiview needs to know which window is active and the name of the interface to send commands to.

--filesystem=host:ro: Must be able to access files and folders to preview them.
--filesystem=/tmp: /tmp is the folder where unrar/unzip and libreoffice put the unzipped/converted file. 
The reason why I put that permission is because from inside flatpak I don't have access to libreoffice and unrar.

example with the python version of Kiview: https://www.youtube.com/watch?v=mCfPZ5dgIZs

Reply:
Let's talk about `/tmp` in https://github.com/flathub/flathub/pull/5138#discussion_r1554638580 thread, it'd be only necessary if you are using flatpak-spawn to launch libreoffice from host.

Flatpak sandbox has its own private tmp.